x-pack/metricbeat/module/activemq/broker/_meta/docs.asciidoc

```diff
@@ -0,0 +1,12 @@
+Broker Metricset retrieves the JMX key `org.apache.activemq:brokerName=*,type=Broker` using Jolokia. It exposes the following metrics:
+
+* `name`: Broker name.
```

Comment:
you don't need to write this list here, as it will be linked automatically from this page. See https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-kafka-consumergroup.html as an example. It links to https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-kafka.html

Reply:
Fixed.